conans/model/ref.py

```diff
@@ -290,8 +290,7 @@ def copy_with_revs(self, revision, p_revision):
         return PackageReference(self.ref.copy_with_rev(revision), self.id, p_revision)
 
     def copy_clear_rev(self):
```

Comment:
Won't be better to call it `copy_clear_prev`? It will it would be less confusing and will help to identify the actual calls to `copy_clear_rev` and analyze if it should be `copy_clear_revs` or `copy_clear_prev`.

Reply:
ok, no problem. I was trying to be consistent with the naming ⚖️

Reply:
Done!